spec/rails_helper.rb

```diff
@@ -62,3 +62,8 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
```

Comment:
Just a reminder for the future -- add configuration to obscure your API key in the url so that your cassettes don't expose your key. For this assessment it's totally fine. Just good security hygiene in general to always include that.